examples/ding-dong.txt

```diff
@@ -0,0 +1 @@
1vrzbBUzrHZn2BzWWO7Nstpvs9zLWenCRi+xNShJqT3akcOr1PeH6fs7GRx6nUiSJEmSJEmSJH0DGbBL2uO+MDXsDVnHofuDwtcyJlY7JZu9wHInUM8yWrUmjQDt+hyn2jOcqkcWGgNONYcsNbvM13sstTZZbKwz31jhVH2ZdrbNTL5Jkx6nais0s03abNCgQ40+WSirADsbhcXZROY9GbJH9la6p3Tf0POwL4EP4xL4iTeZTQbSYaKdO1V4vrPeeAdGc+arkB1KMkoaROoUtBjEGTrxNNtxga24wGa5SGcwy+rwLBvDM6wN5tkctljttVkfNtno19kY1OgWGVv9If1hn15RZ5By+kViUMYD5qjvO6U7Vf0HjAWY+IKFwbokSZIkSZIkSdJeBuySjnRYyDrZSn7cTn5/lXtMiUGZRrO+x5qj+yFZCLTrDVr1BvONWWbqkTOtIYvNAfPNkvlmSbs25FStSzsfcLqxSTvvMJdvMpNt0cq3aWddWvk2zaxHvT6gkffIw5A8FGQU5KEkkMjTgBBL8mz0PAVZKMhCJKVR+p7izrurbhmErHp/o/dbkpNSRiQnpUAKGSlllLFGmdUZxhZlqDGIVSV6EesUwxrD2KJXNhmkNv3Qoj9s000z9Iqx6B6/AAAGyklEQVQ2m+UcG8NTbA3brPZn2C6arPXqbPQz1vo528MamwPYHkJ3mOgXJbtV6ePjWRx8nsLeIP2451uSJEmSJEmSJEkG7JIe0jhInxRGz4/D9v2B7mHt5WNKdIeRfgFbfcizxBchp5Y1aeQ5rUadegg0a5FmnpitJ9q1yFyz4FR9yKn6gIV6j7lGl8XGJqcaW8zVN2nXurRCh0bepxEGNEKPOj2aWZda6tHKOtRD9XOW+gSGpDgkpYJqwHu+90ZOICORUaY6RWhSxJwY6pSxQZGqQL0/nGUrnmYrzbFZzNMZzLI9nGG7aLM1bLNZtNkummz2m3RjTmdQo1fmbA1zOsOMzhC6RaBfQL+MlGVBkaCMJWUMxATlIQn4ZFV61ZB+t8tANDWXJEmSJEmSJEk6EQbskh7ZQfnt5GzvcfP1g4L2mBIxQRF3l4acPAvUe2V1n0MjD7RrGa1axmwjZ7Ze51S9yUKzyVy9zZlWk7nGDPP1GWZqXdpZh0bWp5ENqup2erTyLjWqgL2Rd2nSpRW2aWQdGmmbPKtauZehTkGdmBoUNCmpEWO9CtdpUlCnSPWqaj3UKco6/dSmH2fZLBfYiqdYH8zTKWbYGrbZHrbYHLbYGDboDOtsDnO6o1C9HwOdIfSKRG8I/aJkGBNlnJz8DocOih8d62zygKfgRSlJkiRJkiRJkvQYGLBLOjHj8DyRjsqDp1LuhMxHyYDW6Db/FTtao+HsJ3TcS6vUJUmSJEmSJEmSHrvMQyBJkiRJkiRJkiRJ0oMZsEuSJEmSJEmSJEmSNAUDdkmSJEmSJEmSJEmSpmDALkmSJEmSJEmSJEnSFAzYJUmSJEmSJEmSJEmaggG7JEmSJEmSJEmSJElTMGCXJEmSJEmSJEmSJGkKBuySJEmSJEmSJEmSJE3BgF2SJEmSJEmSJEmSpCkYsEuSJEmSJEmSJEmSNAUDdkmSJEmSJEmSJEmSpmDALkmSJEmSJEmSJEnSFAzYJUmSJEmSJEmSJEmaggG7JEmSJEmSJEmSJElTMGCXJEmSJEmSJEmSJGkKBuySJEmSJEmSJEmSJE3BgF2SJEmSJEmSJEmSpCkYsEuSJEmSJEmSJEmSNAUDdkmSJEmSJEmSJEmSpmDALkmSJEmSJEmSJEnSFAzYJUmSJEmSJEmSJEmaggG7JEmSJEmSJEmSJElTMGCXJEmSJEmSJEmSJGkKBuySJEmSJEmSJEmSJE3BgF2SJEmSJEmSJEmSpCkYsEuSJEmSJEmSJEmSNAUDdkmSJEmSJEmSJEmSpmDALkmSJEmSJEmSJEnSFAzYJUmSJEmSJEmSJEmaggG7JEmSJEmSJEmSJElTMGCXJEmSJEmSJEmSJGkKBuySJEmSJEmSJEmSJE3BgF2SJEmSJEmSJEmSpCkYsEuSJEmSJEmSJEmSNAUDdkmSJEmSJEmSJEmSpmDALkmSJEmSJEmSJEnSFAzYJUmSJEmSJEmSJEmaggG7JEmSJEmSJEmSJElTMGCXJEmSJEmSJEmSJGkKBuySJEmSJEmSJEmSJE3BgF2SJEmSJEmSJEmSpCkYsEuSJEmSJEmSJEmSNAUDdkmSJEmSJEmSJEmSpmDALkmSJEmSJEmSJEnSFAzYJUmSJEmSJEmSJEmaggG7JEmSJEmSJEmSJElTMGCXJEmSJEmSJEmSJGkKBuySJEmSJEmSJEmSJE3BgF2SJEmSJEmSJEmSpCkYsEuSJEmSJEmSJEmSNAUDdkmSJEmSJEmSJEmSpmDALkmSJEmSJEmSJEnSFAzYJUmSJEmSJEmSJEmaggG7JEmSJEmSJEmSJElTMGCXJEmSJEmSJEmSJGkKBuySJEmSJEmSJEmSJE3BgF2SJEmSJEmSJEmSpCkYsEuSJEmSJEmSJEmSNAUDdkmSJEmSJEmSJEmSpmDALkmSJEmSJEmSJEnSFAzYJUmSJEmSJEmSJEmaggG7JEmSJEmSJEmSJElTMGCXJEmSJEmSJEmSJGkKBuySJEmSJEmSJEmSJE3BgF2SJEmSJEmSJEmSpCkYsEuSJEmSJEmSJEmSNAUDdkmSJEmSJEmSJEmSpmDALkmSJEmSJEmSJEnSFAzYJUmSJEmSJEmSJEmaggG7JEmSJEmSJEmSJElTMGCXJEmSJEmSJEmSJGkKBuySJEmSJEmSJEmSJE3BgF2SJEmSJEmSJEmSpCkYsEuSJEmSJEmSJEmSNAUDdkmSJEmSJEmSJEmSpmDALkmSJEmSJEmSJEnSFAzYJUmSJEmSJEmSJEmaggG7JEmSJEmSJEmSJElTMGCXJEmSJEmSJEmSJGkKBuySJEmSJEmSJEmSJE3h/wJakurDI0BDPwAAAABJRU5ErkJggg==
```

Comment:
What is this file? I don't think it should be submitted to the repo?

Reply:
Yes, I will remove it at later commit.